.github/workflows/compute.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           node-version: "20.10.0"
 
       - name: Install dependencies
-        run: yarn
+        run: yarn i --immutable --immutable-cache --check-cache
```

Comment:
Not necessary just force of habit. According to [yarn docs](https://yarnpkg.com/cli/install) it is safer, since it avoids user pushing a yarn.lock that would load different packages, it aborts if anything doesn't match.